src/Microsoft.HttpRepl/Commands/BaseHttpCommand.cs

```diff
@@ -618,5 +618,13 @@ protected override IEnumerable<string> GetOptionValueCompletions(IShellState she
 
             return null;
         }
+
+        public string GetExampleBody(string path, ref string contentType, string method, HttpState httpState)
```

Comment:
Does this need to be public?  It was on HttpState so that it could be used here, but now that it's moved, maybe it can be private?

Reply:
Nope, doesn't need to be public. Changed in 30fe0a3.